bot.js

```diff
@@ -21,8 +21,14 @@ console.log(process.argv);
 
 (require('mineflayer-navigate'))(mineflayer)(bot);
 
+bot.on('message', function(message) {
```

Comment:
Maybe we could add this?

```js
type Message = {
  extra: { text: string }[]
};

bot.on('message', function(message: Message) {
  // ...
});
```
So the type of the callback is known here, even though it's `Function` in the library definition. Wanted to put the type declaration there but apparently that's not supported yet.